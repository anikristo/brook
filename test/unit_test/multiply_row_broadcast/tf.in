2 9
0 1 2 1 2 1 7 -1 6
0 1 2 0 -3 4 4 5 3