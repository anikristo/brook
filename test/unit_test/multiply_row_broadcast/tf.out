2 9
0.0 1.97185 5.99966 3.99999 9.99999 5.99991 49.0 -8.59016 -5.43905
0.0 1.97185 5.99966 0.0 -15.0 23.9996 28.0 42.9508 -2.71953
